iron/distribution.yaml

```diff
@@ -3181,6 +3181,16 @@ repositories:
       url: https://github.com/ros-planning/moveit_visual_tools.git
       version: ros2
     status: maintained
+  mp2p_icp:
```

Comment:
I have a question on how this package is going to interact with https://github.com/ros/rosdistro/blob/5bb368ac3182c13773638f8a9fee63ad1202c1c1/iron/distribution.yaml#L3078 , which is already released.  As far as I understand, this new package has the same name as that package, so you won't be able to release it here.  What is your plan for dealing with this?

Reply:
Hi @clalancette !

Yes, you are correct. Long story short: 
- Current state: one git repo (with submodules) named "mola".
- New state: 3 git repos (no submodules anymore)
  - mola_common : Already merged for indexing in #39230  (Read the "NOTE" section at the end of that PR).
  - mp2p_icp: This PR
  - mola: New releases to be made, now of course without the other two packages above, once they are merged and bloom-released.

If it's an issue to have the same package released twice, even during 1 day, I can create a PR to remove the "release:" section of "mola". 

Thanks, and sorry for the mess, this new state of things makes more sense thinking of the modularity of new forthcoming packages.